app/uk/gov/hmrc/exports/models/declaration/Parties.scala

```diff
@@ -19,31 +19,37 @@ package uk.gov.hmrc.exports.models.declaration
 import play.api.libs.json.{Json, OFormat}
 
 case class EntityDetails(eori: Option[String], address: Option[Address])
+
 object EntityDetails {
   implicit val format: OFormat[EntityDetails] = Json.format[EntityDetails]
 }
 
 case class Address(fullName: String, addressLine: String, townOrCity: String, postCode: String, country: String)
+
 object Address {
   implicit val format: OFormat[Address] = Json.format[Address]
 }
 
 case class ExporterDetails(details: EntityDetails)
+
 object ExporterDetails {
   implicit val format: OFormat[ExporterDetails] = Json.format[ExporterDetails]
 }
 
 case class ConsigneeDetails(details: EntityDetails)
+
 object ConsigneeDetails {
   implicit val format: OFormat[ConsigneeDetails] = Json.format[ConsigneeDetails]
 }
 
-case class DeclarantDetails(details: EntityDetails)
+case class DeclarantDetails(details: Option[EntityDetails])
```

Comment:
Do we have a scenario when we have no declarant details? I thought this is mandatory for the whole declaration, no matter what type it has. How would we be able to tell who has submitted it?

Reply:
DeclarantDetails are optional in Parties. This element is mandatory in every journey except Clearance

Reply:
Besides, in the `ExportsDeclaration` model this field is already optional, so why would we need to have so many nested `Option`s ?

Reply:
That's a good question. As I understand this approach allow us to present empty element on the summary page. The same approach is on the representative details